ui/app/manager/src/index.ts

```diff
@@ -170,6 +151,7 @@ fetch(configURL).then(async (result) => {
     orApp.managerConfig = appConfig.manager;
 
     orApp.appConfigProvider = (manager) => {
+        manager.managerAppConfig = appConfig as unknown as ManagerConf
```

Comment:
This soft cast to `unknown` then to `ManagerConf` I guess is because `ManagerAppConfig` model is missing the `manager` key which should be fixed by comments above in `ManagerConf`